docs/integrations/docker.md

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_position: 69
+title: Docker
+---
+
+
+# Setup a DBOS application with Docker
+
+
+This guide shows you how to setup a starter DBOS application and its Postgres database using docker.
+
+We'll need two files:
+
+## Docker compose
+
+With Docker compose we'll start two containers, one for Postgres and one for a DBOS Python application.
```

Comment:
The Dockerfile should be first, and there should be a little more text emphasizing that it's JUST Python and Postgres, no fancy DBOS-specific stuff.